.github/workflows/ms.network.networkinterfaces.yml

```diff
@@ -0,0 +1,142 @@
+name: 'Network: NetworkInterfaces'
+
+on:
+  workflow_dispatch:
+    inputs:
+      removeDeployment:
+        type: boolean
+        description: 'Remove deployed module'
+        required: false
+        default: true
+      prerelease:
+        type: boolean
+        description: 'Publish prerelease module'
+        required: false
+        default: false
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/actions/templates/**'
+      - '.github/workflows/ms.network.networkInterfaces.yml'
+      - 'arm/Microsoft.Network/networkInterfaces/**'
+      - 'arm/.global/global.module.tests.ps1'
+      - '!*/**/readme.md'
+      - 'utilities/pipelines/**'
+      - '!utilities/pipelines/dependencies/**'
+
+env:
+  variablesPath: 'global.variables.yml'
+  modulePath: 'arm/Microsoft.Network/networkInterfaces'
+  workflowPath: '.github/workflows/ms.network.networkInterfaces.yml'
```

Comment:
Currently all workflow file names are lowercase so I'd suggest to rename the file and then this path for consistency (Remember the change of case only won't be taken as an update. You'd need to update the name in git twice)